libs/ledger-live-common/src/families/bitcoin/js-getTransactionStatus.ts

```diff
@@ -107,7 +110,7 @@ const getTransactionStatus = async (
     `totalSpent ${totalSpent.toString()} amount ${amount.toString()}`
   );
 
-  if (!errors.amount && !amount.gt(0)) {
+  if (!errors.amount && !amount.gt(0) && !opReturnData) {
     errors.amount = useAllAmount
```

Comment:
I guess this modification is not necessary? because even with opReturnData, we should check NotEnoughBalance error.